server/Services/GameRound.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Api.Hubs;
+using Api.Extensions;
+using System.Threading;
+using Microsoft.AspNetCore.SignalR;
+using Api.DTO;
+
+namespace Api.Services
+{
+  public class GameRound
+  {
+    private readonly int EggCards = 10;
+    private readonly int HenCards = 5;
+    private readonly int FoxCards = 5;
```

Comment:
Put the correct number of cards from the official rules (pinned message in Slack):
- 19 eggs
- 10 hens
- 10 [foxes](https://youtu.be/jofNR_WkoCE?t=39)